Yosemite/Yosemite/Model/WooPlans/LegacyWooPlan.swift

```diff
@@ -1,7 +1,165 @@
 import Foundation
 import SwiftUI
 
-public struct WooPlan: Decodable {
+public struct WooPlan: Identifiable {
+    public let id: String
+    public let planFeatures: [String]
+
+    public static func isEssential(_ plan: String) -> Bool {
+        let essentialMonthly = AvailableInAppPurchasesWPComPlans.essentialMonthly.rawValue
+        let essentialYearly = AvailableInAppPurchasesWPComPlans.essentialYearly.rawValue
+
+        if plan == essentialMonthly || plan == essentialYearly {
+            return true
+        } else {
+            return false
+        }
+    }
+
+    public static func loadHardcodedPlan(_ selectedPlanID: String) -> WooPlan {
+
```

Comment:
Not a blocker, but here and in other places in this PR I see there's an empty line after the function/class definition, but AFAIK we have the convention of not leaving that :)

Reply:
Good catch! Removed whitespace in different places with 709d7cea3c21f5ae9403212ee9021f151cbee1e7